recipes/filesdb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "filesdb" %}
+{% set version = "0.2.5" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "83a30a4ce3991bf92c39140a1b2aeec8ed6750942c34a4fffa195f6b035a458b" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - filesdb = filesdb.__init__:main
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python >=3
```

Comment:
can you skip the python3 buidls in the build section please `build/skip: True  # [py2k]`